discovery_engine/lib/src/domain/event_handler.dart

```diff
@@ -268,12 +291,17 @@ class EventHandler {
     Hive.registerAdapter(DurationAdapter());
     Hive.registerAdapter(UriAdapter());
     Hive.registerAdapter(EmbeddingAdapter());
+    Hive.registerAdapter(FeedMarketAdapter());
```

Comment:
❓we don't perists `FeedMarket` anymore, should I revert changes to this class so that Hive type adapters are not generated and we don't register it, or should we leave it, cause it might be usefull in the future and it doesn't break anything now?

Reply:
i guess we can leave it for now. if it's not needed in the future, it could be reverted in a follow up pr.